package.json

```diff
@@ -27,6 +27,7 @@
     "jsdom": "^3.1.2",
     "lunr": "^0.5.9",
     "mocha": "^2.2.4",
+    "object-assign": "^2.0.0",
```

Comment:
needed for -> "test-browser": "browserify test/browser/index.js -t babelify --outfile test/browser/tests.js"